### ISSUE TITLE
set_subplots

### ISSUE THREAD

**Issue by nicolaskruchten** [plotly/plotly.py#2274]
I think it would be nice to be able to do something like `fig = go.Figure(); fig.set_subplots(...)` as an alternative to `make_subplots()` so as to save an import.

It would work maybe only once, if nothing is already there? Maybe later it can work if something is already there so long as there's no conflict or something.

**Comment by emmanuelle** [comment]
probably `fig.set_subplots()` would call directly `make_subplots` and use the `data` and `layout` of the figure returned by `make_subplots`?

**Comment by emmanuelle** [comment]
For people interested in contributing this feature, this would have to be implemented in this part https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/codegen/figure.py#L19 of the library (the `codegen`).

**Comment by nicolaskruchten** [comment]
> probably fig.set_subplots() would call directly make_subplots and use the data and layout of the figure returned by make_subplots?

Something like that I think... Maybe just try to merge the layout and abort if there is a conflict?

**Comment by gvwilson** [comment]
Hi - we are currently trying to tidy up Plotly's public repositories to help us focus our efforts on things that will help users most. Since this issue has been sitting for several years, I'm going to close it. If it's still a concern, we'd be grateful if you could open a new issue (with a short reproducible example if appropriate) so that we can add it to our backlog. Thanks for your help - @gvwilson
